src/test/scala/org/broadinstitute/hail/utils/OrderedRDDSuite.scala

```diff
@@ -40,7 +43,9 @@ class OrderedRDDSuite extends SparkSuite {
         Iterator((partitionIndex, sorted, correctPartitioning, if (a.nonEmpty) Some((a.head._1, a.last._1)) else None))
```

Comment:
Usually when I see a 4-tuple, I think it should become a class. If nothing more than to give names to fields.

```scala
case class PartitionSummary(partitionIndex: Int,
  sorted: Boolean, 
  correctPartitioning: Boolean,
  maybeBounds: Option[(Int, Int)])
```